home.html

```diff
@@ -0,0 +1,139 @@
+    </div>
+      
+    </nav>
+  </header>
+    <main>
+        <div class="introduction">
+        <p class="topic">Welcome to sport Festival</p>
+        <h2 class="topic2">Sport Festival 2022</h2>
+        <p class="main-paragraph">Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia ipsum aut distinctio aspernatur exercitationem? Cum quis facere natus accusantium perspiciatis, fugit at quasi in excepturi officia eaque distinctio nemo beatae? Lorem ipsum, dolor sit amet consectetur adipisicing elit. Vero at dolorum neque iusto sit deserunt excepturi repellat porro dolor omnis quas, tempora magni nam. Unde eligendi consequuntur placeat totam ipsa?</p>
+        <div class="footer">
+        <p class="date">2023.07.19(THU) <p>
+           <p class="address"> @ Macroyan, Kabul, Afghanistan</p>
+        </div>
+    </div>
+    </main>
+
+    <h2 class="background-img2"></h2>
+    <div class="card-imgs">
+     <div class="main-sports">    
+    <h1 class="main-sport">Main sports</h1>
+    <img class="bottom-line" src="images/Underline.png" alt="underline image">
+    </div>
+    <div class="cards"> 
+    <div class="card">
+    <img class="card-img-top" src="images/basketball.jpg" alt="Card image cap">
+    <div class="sports-name">
+        <h2 class="sport-name name-1">basketball</h2></div> 
+    <div class="card-body">
+    <p class="card-text text-1">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+   
+    </div>
+    </div>
+    <div class="card">
+    <img class="card-img-top" src="images/Football.jpg" alt="Card image cap">
+    <div class="sports-name">
+        <h2 class="sport-name name-2">Football</h2></div> 
+    <div class="card-body">
+    <p class="card-text text-2">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+   
+    </div>
+</div>
+    <div class="card">
+    <img class="card-img-top" src="images/cricket.jpg" alt="Card image cap">
+    <div class="sports-name">
+    <h2 class="sport-name name-3">Cricket</h2></div> 
+    <div class="card-body">
+    <p class="card-text text-3">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+    
+    </div>
+</div>
+    <div class="card">
+    <img class="card-img-top" src="images/Weighting.jpg" alt="Weigth liftting">
+    <div class="sports-name name-4">
+        <h2 class="sport-name">Weight lefting</h2></div> 
+    <div class="card-body">
+    <p class="card-text text-4">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+    </div>
+    </div>
+</div>
+    <button class="button1" href="./" type="submit">Join Sports Galla 2022</button>
+    <a class="button2" href="./" type="submit">SEE THE WHOLE PROGRAM</a>
+    </div>
+    </div>
+   
+ 
+    <section class="background-img3" id="background-img3">
+
+
+    </section>
+ 
+
+   <div class="background-img4">
```

Comment:
- [ ] Please remove all the trailing spaces between your codes in order to keep it clean 😉